spec/models/manageiq/providers/hawkular/middleware_manager/event_catcher/stream_spec.rb

```diff
@@ -125,7 +198,7 @@
     let!(:db_server) do
       ems_hawkular.middleware_servers.create(
         :feed       => 'f1',
-        :ems_ref    => '/t;hawkular/f;f1/r;resource1',
```

Comment:
`ems_ref` is the primary key in MiQ (isolated to each provider)... What will happen if two independent servers (aka, two agents) happen to have the same server `id`??? Having the feed in the ems_ref worked, to some extent, to avoid collisions. But, now, is it just a bare ID???

Previously, potential collisions happened for the _domain_ case, where `ems_ref` included the server name instead of a random guid like in the case of standalone servers. So, if you have two domains each one with a server named `server-one` a collision in MiQ was being avoided by having the feed id in `ems_ref`. How would that be handled now?... May collisions like this never happen?

I haven't seen how the IDs look like, so I'm unsure.

Reply:
Ids are opaque (kind of), and unique, so it must be fine. In the inventory server, it's the primary key for resources, there's no composed key. And in its implementation it contains the feedid.

Reply:
You can assume that IDs are unique. Currently implementation uses the old "CanonicalPath" as their "id".

Reply:
https://github.com/ManageIQ/manageiq-providers-hawkular/pull/75/files#diff-a6822e8aa662958bb0ab11a0ec7cc558R23
and
https://github.com/ManageIQ/manageiq-providers-hawkular/pull/75/files#diff-a6822e8aa662958bb0ab11a0ec7cc558R111

Reply:
Ok, I'm happy if IDs are unique :)